src/components/LuceneQueryEditor.tsx

```diff
@@ -41,6 +40,12 @@ export function LuceneQueryEditor(props: LuceneQueryEditorProps){
     if (!word){ return null }
       suggestions = await autocompleter(word?.text);
     if (suggestions && suggestions.options.length > 0 ) {
+      // Fixes autocompletion inserting an extra quote when the cursor is before a quote
```

Comment:
does it make sense to only check that last character? 
I don't understand the slice with `context.pos`. Can you explain exactly what we want to check here?

Reply:
> does it make sense to only check that last character?
I've modified it to check for all characters following the cursor for a double quote instead of just the immediate character.

`"quickwit-indexing/src/actors/indexing_pipeline.rs"`

Some examples of autocompletion where ↑ is the current cursor position.:
1. Example of completing immediately behind a double quote: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/↑"`
- The returned completion would **omit** the final quote `"quickwit-indexing/src/actors/indexing_pipeline.rs`.
- Applying the completion would give us: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/indexing_pipeline.rs"`
- The current behaviour would instead give us: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/indexing_pipeline.rs""`

2. Example of completing without quotes: `events.event_attributes.code.filepath:↑`
- The returned completion would be **surrounded** by quotes `"quickwit-indexing/src/actors/indexing_pipeline.rs"`.
- Applying the completion would give us: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/indexing_pipeline.rs"`
3. Example of completing with whitespace before double quotes: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/↑           "`
- The returned completion would **omit** the final quote `"quickwit-indexing/src/actors/indexing_pipeline.rs`.
- Applying the completion would give us: `events.event_attributes.code.filepath:"quickwit-indexing/src/actors/indexing_pipeline.rs           "` **which isn't ideal** but is better than an extra double quote